app/src/main/java/com/github/fribourgsdp/radio/QuitGameOrLobbyDialog.kt

```diff
@@ -0,0 +1,40 @@
+package com.github.fribourgsdp.radio
+
+import android.app.Activity
+import android.content.Context
+import android.content.ContextWrapper
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.core.os.bundleOf
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.setFragmentResult
+
+
+class QuitGameOrLobbyDialog(val ctx: Context, private val isHost: Boolean, private val isFromGameActivity: Boolean, private val user: User, private val db: Database, private val gameID: Long): DialogFragment() {
```

Comment:
What's the point of giving `db: Database` and `gameID` here ?
I don't think you are still using it.

Reply:
You are right, this was from a previous implementation.